js/fibonacci.js

```diff
@@ -0,0 +1,178 @@
+//Deceleration of HTML Elements
+const elementOfIndex = document.getElementById("num1");
+const calculateButton = document.getElementById("btn1");
+const elementOfResult = document.getElementById("num2");
+const loadingSpinner = document.getElementById("spinner");
+const validFeedback = document.getElementById("validationFeedback1");
+const calcRes = document.getElementById("resultsOfCalculations");
+const saveCalc = document.getElementById("save-calculation");
+const sortSelectElement = document.getElementById("sort-results");
+
+// =================================================================
+
+//On a refresh of the page, the list of results will be displayed
+getFibonacciResults().then(arrayOfResponses => sortResults(arrayOfResponses)).then(arrayOfResponses => displayResults(arrayOfResponses));
```

Comment:
sortResults() & displayResults() are both syncronus functions, so .then() is enough. Also, whenever you are using .then() you should have a catch block to handle possible errors.